cmd/devtool/README.md

```diff
@@ -0,0 +1,29 @@
+# devtool
+
+An on-chain workflow testing tool that supports the following:
+
+- Automatically submitting the necessary setup transactions for each node type
+- Generating a Bash script with default CLI flags to start each node type
+
+This tool requires access to an ETH node connected to private network.
+
+## Setting up a private ETH network
+
+```
+docker pull livepeer/geth-with-livepeer-protocol
```

Comment:
it should be `livepeer/geth-with-livepeer-protocol:pm`

Reply:
Fixed in 90b4061